news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,321 @@
+
+I also think that my score in [amazarashi - Sayonara Gokko (TV Size) \[_Epreus' Light Insane\]](https://osu.ppy.sh/scores/1948389545) is worth mentioning, because I remember very well that in that day it was very cold, and I was very sleepy already (it was very late night), but I really wanted to set this FC. My hands were freezing, but after about 40 minutes, I did it!
+
+![](/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/sayonara_gokko.jpg)
+
+I was undecided in what else to mention, but I think I will go with [Frums (unknown "lambda") - 19ZZ \[wispy's Insane\] with HDHR](https://osu.ppy.sh/scores/1962037812), from almost 1 year ago. This map is a very hard one to FC and especially to SS, due some weird sliders. As far as I remember, I checked how the map was before it was even ranked and practiced a bit, and I didn't have hope to even being able to FC this at all, but ended up making the FC and SS after about 23 minutes after ranked! I was really impressed by myself, and I think my HR skill peaked there, because since then my HR performance started to drop month by month... (possibly because I started to grind SS harder, so I was overall more tired etc.).
+
+![](/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/19ZZ.jpg)
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+If you could choose any osu! player, who would you want to meet IRL the most, and why?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+Wow, I'll be honest that this question is the hardest one for me! I am very shy, so I never thought so much about this. I could mention any of the close friends I have, but that would be way too easy (or too hard, because other friends not mentioned could get sad '-'). 
+
+Anyways, I feel there are many players that would be interesting to meet IRL, I think it one obvious one would be ExGon, because I really admire him and is by far my biggest inspiration to this day! A less obvious one that I'd like to meet one day (and maybe a unexpected one, since he's not from osu!catch world) is [BTMC](https://osu.ppy.sh/users/3171691), since from time to time I enjoy watching his livestreams and YouTube videos and he's a great std player. He seems to be really fun! But in both cases, I would barely have the courage to even ask for a photo...
```

Comment:
```suggestion
Anyways, I feel there are many players that would be interesting to meet IRL, I think it one obvious one would be ExGon, because I really admire him and is by far my biggest inspiration to this day! A less obvious one that I'd like to meet one day (and maybe a unexpected one, since he's not from osu!catch world) is [BTMC](https://osu.ppy.sh/users/3171691), since from time to time I enjoy watching his livestreams and YouTube videos and he's a great osu! player. He seems to be really fun! But in both cases, I would barely have the courage to even ask for a photo...
```